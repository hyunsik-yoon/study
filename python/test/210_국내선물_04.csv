거래일자,거래번호,거래종류,적요명,종목번호,종목명,수량,단가,거래금액,수수료,예수금전잔,예수금금잔,정산금액,약정금액,세금합계,대용잔고,기타수표금액,미수/변제
2023-12-04,3,매수,당일차금매수입금                                                                ,foo,bar,120,"1,302.30","143,000","7,800","43,713,841","43,849,041","135,200","169,299,000",0,0,0,0
2023-12-05,1,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"286,000",0,"43,755,491","43,469,491","286,000",0,0,0,0,0
2023-12-06,1,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"1,001,000",0,"41,489,491","42,490,491","1,001,000",0,0,0,0,0
2023-12-07,4,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"143,000",0,"43,635,491","43,778,491","143,000",0,0,0,0,0
2023-12-08,1,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"1,417,000",0,"43,778,491","45,195,491","1,417,000",0,0,0,0,0
2023-12-11,4,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"2,093,000",0,"46,555,491","44,462,491","2,093,000",0,0,0,0,0
2023-12-12,2,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"1,261,000",0,"45,337,491","46,598,491","1,261,000",0,0,0,0,0
2023-12-13,2,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"286,000",0,"46,371,971","46,085,971","286,000",0,0,0,0,0
2023-12-14,2,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"663,000",0,"45,510,971","46,173,971","663,000",0,0,0,0,0
2023-12-15,3,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"3,029,000",0,"46,863,711","43,834,711","3,029,000",0,0,0,0,0
2023-12-18,3,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"26,000",0,"43,984,711","44,010,711","26,000",0,0,0,0,0
2023-12-19,4,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"52,000",0,"45,985,711","46,037,711","52,000",0,0,0,0,0
2023-12-20,3,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"1,313,000",0,"48,387,711","49,700,711","1,313,000",0,0,0,0,0
2023-12-21,2,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"949,000",0,"48,925,711","47,976,711","949,000",0,0,0,0,0
2023-12-22,2,입금,매수갱신차금입금                                                                ,foo,bar,120,0,"676,000",0,"46,951,071","47,627,071","676,000",0,0,0,0,0
2023-12-26,2,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"299,000",0,"48,277,071","47,978,071","299,000",0,0,0,0,0
2023-12-27,2,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"1,001,000",0,"47,778,071","46,777,071","1,001,000",0,0,0,0,0
2023-12-28,3,출금,매수갱신차금출금                                                                ,foo,bar,120,0,"78,000",0,"48,527,071","48,449,071","78,000",0,0,0,0,0
