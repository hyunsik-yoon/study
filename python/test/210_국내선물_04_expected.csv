종목명,진입일자,진입단가,청산일자,청산단가,거래수량,총수수료,총세금합계,순수익
bar,12/04/2023,"1,302.30",not yet,"-1,234,567,890",120,"7,800",0,-1326000
